Sources/Card/Models/CardExpiry.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+public struct CardExpiry: Codable {
```

Comment:
```suggestion
public struct ExpirationDate: Codable {
```

We can remove the `Card` prefix since it lives in the Card module. Also `ExpirationDate` is more standard, at least in the docs I've seen for BT and PP.